specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2021-10-01/AlertRules.json

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
Did you take the version from 2021-03-01-preview?
because it seems that the enrichment models are missing

Reply:
No, this version is based on the previous Stable.
You'll have to add any changes to the previous Stable after this PR is merged.